hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OFSPath.java

```diff
@@ -174,4 +185,27 @@ public boolean isRoot() {
   public boolean isVolume() {
     return this.getBucketName().isEmpty() && !this.getVolumeName().isEmpty();
   }
+
+  /**
+   * Get the bucket name of temp for given username.
+   * @param username Input user name String. Mustn't be null.
+   * @return Username MD5 hash in hex digits.
+   */
+  @VisibleForTesting
+  static String getTempMountBucketName(String username) {
+    Preconditions.checkNotNull(username);
+    // TODO: Improve this to "slugify(username)-md5(username)" for better
+    //  readability?
+    return DigestUtils.md5Hex(username);
```

Comment:
Can we use the name directly to make it more user friendly?

Reply:
As discussed, since other users' `tmp` buckets aren't visible to current user via `ofs://`. This could be addressed when the requirement of accessing other users' temp bucket happen.
And there are several related future improvements. Like:
1. allowing users to access all volumes (that they have access to) via a special reserved path (e.g. `ofs://om/.volumes/<volume_name>/..` or `ofs://om/.reserved/..`). Also, admins should be able to list all volumes, not just the ones they own.
2. Provide a utility in Ozone Shell to access the users' temp buckets conveniently for admins.